lib/IAMClient.js

```diff
@@ -76,6 +77,7 @@ class VaultClient {
         this.log = new this.logApi.Logger('VaultClient');
         this._path = path;
         this.useAuthenticatedAdminRoutes = false;
+        this.parameterValidation = parameterValidation !== undefined ? parameterValidation : true;
```

Comment:
If we want to enforce a boolean here
```suggestion
        this.parameterValidation = parameterValidation !== undefined ? !!parameterValidation : true;
```